rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -437,9 +437,9 @@ def get_or_decode_transaction_events(
                     (transaction.tx_hash, serialized_chain_id, HISTORY_MAPPING_STATE_DECODED),
                 )
                 if cursor.fetchone()[0] != 0:  # already decoded and in the DB
-                    events = self.dbevents.get_history_events(
+                    events: list['EvmEvent'] = self.dbevents.get_history_events(  # type: ignore[assignment]  # noqa: E501
```

Comment:
The type:ignore seems to have still not been addressed here as mentioned in this comment: https://github.com/rotki/rotki/pull/5726#discussion_r1138543311